demos/twa-notification-delegation-orientation/README.md

```diff
@@ -0,0 +1,8 @@
+# Trusted Web Activity / Notification Delegation Demo
```

Comment:
How come the demo you've made is called `twa-notification-delegation-orientation`? It looks like you've mostly copied the notification delegation demo (which is fine), but not got rid of the notification delegation parts.

What I'd recommend is that you copy from the `twa-basic` instead (it's the simplest one) and then name it `twa-landscape` or `twa-orientation` or something like that.

Reply:
Yeah. Andre said we do not want to just modify the twa-basic demo as some users may not want to touch with orientation. So I copied the content from the notification to support orientation. 

The reason I chose the notification one is that the website for the twa-basic (svgomg) is not working now.